sdk/metricsadvisor/azure-ai-metricsadvisor/src/samples/README.md

```diff
@@ -21,38 +20,100 @@ Key concepts are explained in detail [here][SDK_README_KEY_CONCEPTS].
+|[ListSeriesDefinitionsForMetricAsyncSample][list_series_def_sample]
+& [ListSeriesDefinitionsForMetricAsyncSample][list_series_def_async_sample] | List series definition for a metric|
+|[ListIncidentsDetectedSample][list_incidents_sample]
+& [ListIncidentsDetectedAsyncSample][list_incidents_async_sample] | List incidents detected by a detection configuration|
 
 ## Troubleshooting
 Troubleshooting steps can be found [here][SDK_README_TROUBLESHOOTING].
 
 ## Next steps
-TODO
-Check out the [API reference documentation][java_fr_ref_docs] to learn more about
+Check out the [API reference documentation][java_ma_ref_docs] to learn more about
 what you can do with the Azure Metrics Advisor client library.
-Following section provides code samples for some of the advanced scenarios in Metrics Advisor Client library:
-|**Advanced Sample File Name**|**Description**|
-|----------------|-------------|
-
 
 ## Contributing
 If you would like to become an active contributor to this project please refer to our [Contribution
 Guidelines][SDK_README_CONTRIBUTING] for more information.
 
 <!-- LINKS -->
-[SDK_README_CONTRIBUTING]: ../../README.md#contributing
-[SDK_README_GETTING_STARTED]: ../../README.md#getting-started
-[SDK_README_TROUBLESHOOTING]: ../../README.md#troubleshooting
-[SDK_README_KEY_CONCEPTS]: ../../README.md#key-concepts
-[SDK_README_DEPENDENCY]: ../../README.md#include-the-package
-[SDK_README_NEXT_STEPS]: ../../README.md#next-steps
-[java_fr_ref_docs]: https://aka.ms/azsdk-java-metricsadvisor-ref-docs
+[SDK_README_CONTRIBUTING]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/metricsadvisor/azure-ai-metricsadvisor#contributing
+[SDK_README_GETTING_STARTED]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/metricsadvisor/azure-ai-metricsadvisor#getting-started
+[SDK_README_TROUBLESHOOTING]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/metricsadvisor/azure-ai-metricsadvisor#troubleshooting
+[SDK_README_KEY_CONCEPTS]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/metricsadvisor/azure-ai-metricsadvisor#key-concepts
+[SDK_README_DEPENDENCY]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/metricsadvisor/azure-ai-metricsadvisor#include-the-package
+[SDK_README_NEXT_STEPS]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/metricsadvisor/azure-ai-metricsadvisor#next-steps
+[java_ma_ref_docs]: https://aka.ms/azsdk-java-metricsadvisor-ref-docs
+[data_feed_sample]: https://github.com/Azure/azure-sdk-for-java/blob/70706988e48f0f1fcfa5fc8910958924d9d29b87/sdk/metricsadvisor/azure-ai-metricsadvisor/src/samples/java/com/azure/ai/metricsadvisor/administration/DatafeedSample.java
+[data_feed_async_sample]:https://github.com/Azure/azure-sdk-for-java/blob/70706988e48f0f1fcfa5fc8910958924d9d29b87/sdk/metricsadvisor/azure-ai-metricsadvisor/src/samples/java/com/azure/ai/metricsadvisor/administration/DatafeedAsyncSample.java
```

Comment:
Will open a PR to fix these links, once this is merged. Changed the naming of files in this PR and used commit hash to fix the broken link issue.

cc: @anuchandy